foursight_core/react/api/auth.py

```diff
@@ -18,9 +22,19 @@ def __init__(self, auth0_client: str, auth0_secret: str, envs: Envs):
         self._auth0_client = auth0_client
         self._auth0_secret = auth0_secret
         self._envs = envs
+        # acquired from identity or env variable locally
+        self._redis = RedisBase(create_redis_client(
+            url=os.environ['REDIS_HOST'])
+        ) if 'REDIS_HOST' in os.environ else None
 
     _cached_aws_credentials = {}
 
+    def get_redis_handle(self):
```

Comment:
Does this function get used? I see you accessing `._redis`. Why not use an `@property` named `redis` (or `rdis_handler`) and then refer to `self.redis` (or `self.redis_handler`?
```suggestion
    @property
    def redis(self):
```

Reply:
Using `redis` alone may cause confusion as it the name of a library as well. I think use of `@property` makes sense though.